dotcom-rendering/src/types/configContext.ts

```diff
@@ -11,10 +11,12 @@ export type Config =
 	| {
 			renderingTarget: Extract<RenderingTarget, 'Web'>;
 			darkModeAvailable: boolean;
+			inAdvertisingPartnerABTest: boolean;
 			assetOrigin: AssetOrigin;
 	  }
 	| {
 			renderingTarget: Extract<RenderingTarget, 'Apps'>;
 			darkModeAvailable: boolean;
+			inAdvertisingPartnerABTest: boolean;
```

Comment:
I'm not entirely sure how AB tests work on Apps. This could be fine, or it could always be `false`, or worse yet it may not be consistent for a given user.

We could investigate deeper into what happens, but I would suggest using `false` here unless you are really expecting to retrieve testing data from Apps.

Reply:
I am not entirely sure as well but we would be fine with the AB test for Web just to make sure the design is what we expect when the labels are live. I will add `false` to `render.article.apps.tsx`.